### ISSUE TITLE
Create Build System for WeVote.me

### ISSUE THREAD

**Issue by DaleMcGrew** [wevote/WebApp#140]
@nf071590 and @josephevans have discussed refining our build system for posting WebApp on the live WeVote.me servers.

**Comment by DaleMcGrew** [comment]
Hello @josephevans and @nf071590 I have posted the latest code on https://wevote.me and it is ready for Joe to take a look. Here is the Pull Request where Nick mentions `npm run prod` and which shows the files he changed: https://github.com/wevote/WebApp/pull/162

Joe, I think the task is to use `npm run prod` when the "wevote-www" service is started.

I reverted the production-server-specific changes to /home/wevote/WebApp/Gulpfile.js, so the code on the server matches the develop branch. 

@nf071590 is there anything else Joe should know about using  `npm run prod`?
